src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -283,26 +285,29 @@ protected override TDeclarationNode AddNamespace<TDeclarationNode>(TDeclarationN
         {
             var currentParameterList = destination.GetParameterList();
 
-            if (currentParameterList == null)
```

Comment:
this is the thrust of the change.  instead of directly adding hte new parameters to the method syntax here, we instead call through teh helper service which will properly wrap/indent them.

Reply:
the same change happens for VB below